.github/workflows/elixir-dev.gen.yml

```diff
@@ -0,0 +1,248 @@
+# This file was generated. See https://daggerverse.dev/mod/github.com/shykes/gha
+name: elixir-dev
+"on":
+    push:
+        branches:
+            - main
+    pull_request:
+        types:
+            - opened
+            - reopened
+            - synchronize
+            - ready_for_review
+    workflow_dispatch: {}
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
The `github.workflow` is a workflow name. So then how is it important to keep this because generated workflow has only single job?

Reply:
It's to control concurrency of multiple instances of the workflow. Typically if you push 10 times to the same PR, that triggers 10 different instances of the worfklow. With the concurrency group, only 1 instance will run at a time.

Reply:
Ahh, I see. Thanks for the info.